benchmark-service/deployment.yaml

```diff
@@ -37,13 +37,11 @@ spec:
 {% if deploy %}
           - name: HAIL_BENCHMARK_BUCKET_NAME
             value: hail-test
+          - name: START_POINT
```

Comment:
I don't understand why this change would fix the behavior we were seeing. Can you explain?

Reply:
It wouldn't, I just thought it was repetitive to have {% if deploy %} twice

Reply:
Ok. There's still a problem with START_POINT. The deployment logs says this is the START_POINT:

```
2020-11-03 22:03:59,472 start point is 2020-11-02T21:38:53Z
```